src/main/java/io/confluent/connect/jdbc/DataConverter.java

```diff
@@ -51,10 +51,17 @@
 
   public static Schema convertSchema(String tableName, ResultSetMetaData metadata)
       throws SQLException {
+    return convertSchema(tableName, null, metadata);
```

Comment:
Why the addition of the `columnName` to the other version of this method and passing `null` here? As far as I can tell, this seems like an unused addition.